udata/core/spatial/models.py

```diff
@@ -8,7 +8,7 @@
 from werkzeug import cached_property
 
 from udata.app import cache
-from udata.i18n import gettext as _, get_locale, language
+from udata.i18n import gettext as _, lazy_gettext as L_, get_locale, language
```

Comment:
Is `L_` a known convention? I'd be more comfortable with `l_` since it's a function.

Reply:
Yes, it is, already used for [`ngettext`](https://github.com/opendatateam/udata/blob/master/udata/i18n.py#L109), [`pgettext`](https://github.com/opendatateam/udata/blob/master/udata/i18n.py#L114). 
I can make the same for `lazy_gettext`, ie. define it permanently into `i18n.py` instead of using `import .. as`